### PR TITLE
Run all tests in current file

### LINKED ISSUE #16: Run all tests of the currently opened file
Create a command that opens vs-code terminal and run a `mix test "path-of-the-current-file"` on the root folder

### PR DESCRIPTION
**Pull request by GuSilverFlame**
this allows you to run a command and execute all tests in the currently open test file
closes #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,14 +16,19 @@
...
   "main": "./extension.js",
   "contributes": {
     "commands": [
       {
         "command": "extension.elixirJumpToTest",
         "title": "Elixir test: Jump"
+      },
+      {
+        "command": "extension.elixirRunTestFile",
+        "title": "Elixir test: run all tests on file"
```
Comment (samuelpordeus) [inline package.json]:
```suggestion
        "title": "Elixir test: Run all tests on file"
```

Just to keep the pattern 😬

File: extension.js [hunk 2]
```diff
@@ -87,6 +87,32 @@ function activate(context) {
...
+      const isTestFile = openedFilename.includes("_test.exs");
+      const isUmbrella = openedFilename.includes("/apps/");
+
+      if (isTestFile === true) {
+        const testPathFilter = isUmbrella ? /.*\/(apps\/.*)$/ : /.*\/(test\/.*)$/
+        let terminal = vscode.window.createTerminal()
+        terminal.sendText(`mix test ${openedFilename.match(testPathFilter)[1]}`)
+        terminal.show()
+      } else {
+        vscode.window.showInformationMessage(`the current file is not a test file`);
```
Comment (samuelpordeus) [inline extension.js]:
```suggestion
        vscode.window.showInformationMessage(`The current file is not a test file.`);
```

Just to keep aesthetics ✨

File: extension.js [hunk 3]
```diff
@@ -87,6 +87,32 @@ function activate(context) {
...
+      const openedFilename = activeFile.document.fileName;
+
+      const isTestFile = openedFilename.includes("_test.exs");
+      const isUmbrella = openedFilename.includes("/apps/");
+
+      if (isTestFile === true) {
+        const testPathFilter = isUmbrella ? /.*\/(apps\/.*)$/ : /.*\/(test\/.*)$/
+        let terminal = vscode.window.createTerminal()
+        terminal.sendText(`mix test ${openedFilename.match(testPathFilter)[1]}`)
+        terminal.show()
```
Comment (samuelpordeus) [inline extension.js]: What happens if the terminal was already opened? 🤔

Comment (GuSilverFlame) [reply extension.js]: before it would open a new one, now it uses the current one

**Pull request merged**